avaframe/com1DFAPy/com1DFA.py

```diff
@@ -47,37 +50,46 @@
 log.info('MAIN SCRIPT')
 log.info('Current avalanche: %s', avalancheDir)
 
-cfg = cfgUtils.getModuleConfig(DFAtls)['GENERAL']
-cfgFull = cfgUtils.getModuleConfig(DFAtls)
+# Load configuration
+cfg = cfgUtils.getModuleConfig(DFAtls)
+cfgGen = cfg['GENERAL']
 
+# for timing the sims
 startTime = time.time()
+
+
+#+++++++++Inputs++++++++++++++++++++++++
 # ------------------------
-# fetch input data
-demFile, relFiles, entFiles, resFile, flagEntRes = gI.getInputData(avalancheDir, cfgFull['FLAGS'], flagDev=True)
+# fetch input data - dem, release-, entrainment- and resistance areas
+demFile, relFiles, entFiles, resFile, flagEntRes = gI.getInputData(avalancheDir, cfg['FLAGS'], flagDev=True)
```

Comment:
I'd prefer to have the flagDev either:
- as feature flag in the input source directly
or
- in the cfg directly

Reply:
Introduced a featureFlag called featLF in the source file